### ISSUE TITLE
Encoding issue with configuration file

### ISSUE THREAD

**Issue by YannRobert** [schwabdidier/GazePlay#84]
On Windows 10

* Delete the GazePlay directory in the user home directory

* Then run the tobii setup
it creates the GazePlay directory in the user home directory, with a ```GazePlay.properties``` file containing 
```
EYETRACKER=tobii
LANGUAGE=fra
FILEDIR=C:\Users\user\GazePlay\files\
FixationLength=500
CssFile=data/stylesheets/main-green.css


```

This file is using ANSI encoding.

* Then, launch the gazeplay game

It fails at startup with following error

```
2017-11-12 23:47:30,231 INFO  - ***********************
2017-11-12 23:47:30,241 ERROR - Exception in Application start method
2017-11-12 23:47:30,259 ERROR - uncaughtException
java.lang.RuntimeException: Exception in Application start method
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$154(LauncherImpl.java:182)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: Malformed \uxxxx encoding.
        at java.util.Properties.loadConvert(Unknown Source)
        at java.util.Properties.load0(Unknown Source)
        at java.util.Properties.load(Unknown Source)
        at gaze.configuration.Configuration.<init>(Configuration.java:44)
        at net.gazeplay.GazePlay.start(GazePlay.java:57)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$161(LauncherImpl.java:863)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$174(PlatformImpl.java:326)
        at com.sun.javafx.application.PlatformImpl.lambda$null$172(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$173(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$null$147(WinApplication.java:177)
        ... 1 common frames omitted
```

**Comment by YannRobert** [comment]
working on a fix right now

**Comment by schwabdidier** [comment]
If I understand well, I should release a new tobii-setup, isn't it ?

**Comment by YannRobert** [comment]
Yes, the tobii-setup need a new release.

By the way, the tobii-setup is included in the ```gazeplay-dist-1.0.x.zip```, so users just have to click ```tobii-setup.bat```
